packages/graphql-modules/tests/providers.spec.ts

```diff
@@ -11,6 +11,66 @@ import {
 import { makeExecutableSchema } from '@graphql-tools/schema';
 import { parse, execute } from 'graphql';
 
+test('No error in case of module without providers', async () => {
+  @Injectable({
+    scope: Scope.Operation,
+  })
+  class Data {
+    lorem() {
+      return 'ipsum';
+    }
+  }
+
+  const mod = createModule({
+    id: 'lorem',
+    typeDefs: gql`
+      type Query {
+        lorem: String
+      }
+    `,
+    resolvers: {
+      Query: {
+        lorem(
+          _parent: {},
+          _args: {},
+          { injector }: GraphQLModules.ModuleContext
+        ) {
+          return injector.get(Data).lorem();
+        },
+      },
+    },
+  });
```

Comment:
No providers

Reply:
@dotansimha

Reply:
weird. it failed for me :X maybe it's something with alpha-2 and it was already fixed?

Reply:
I will release a patch tomorrow